app/interactors/issue_cert.rb

```diff
@@ -2,5 +2,5 @@ class IssueCert
   include Interactor::Organizer
   include FailOnError
 
-  organize RefreshDomain, AuthorizeRequest, ObtainCert, Log
+  organize RefreshDomain, AuthorizeRequest, ObtainCert
```

Comment:
Did you mean to include logging here?

Reply:
i put the logging on each action that needs it via the concern (RefreshDomain is more of a system/implementation detail)